x-pack/platform/plugins/private/reporting/public/management/components/license_prompt.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPageTemplate,
+  EuiSpacer,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { useKibana } from '@kbn/reporting-public';
+
+const title = (
+  <h2 data-test-subj="license-prompt-title">
+    {i18n.translate('xpack.reporting.schedules.licenseCheck.title', {
+      defaultMessage: `Upgrade your license to use Machine Learning`,
```

Comment:
Should this be
```suggestion
      defaultMessage: `Upgrade your license to use Scheduled Reporting`,
```

If so, then please double-check the title casing of `Scheduled Reporting`.

Reply:
Scheduled Exports.  We are using Exports across the board to be consistent

Reply:
[Updated](https://github.com/elastic/kibana/pull/224849/commits/ea44ff44ab06707bb146afad35ea0fb4b888385c)